### PR TITLE
add option to shorten prompt print in log

### PR DESCRIPTION
**Pull request by leiwen83**
For now, model get big context support, and there is scenario that prompt would be very long, if we also want to kept a eye over the incoming traffic, then there will be huge log created.

### CODE REVIEW

File: vllm/engine/async_llm_engine.py [hunk 1]
```diff
@@ -243,10 +243,12 @@ def __init__(self,
                  *args,
                  log_requests: bool = True,
                  start_engine_loop: bool = True,
+                 max_log_len: int = None,
```
Comment (zhuohan123) [inline vllm/engine/async_llm_engine.py:246]: Nit: order
```suggestion
                 log_requests: bool = True,
                 max_log_tokens: int = None,
                 start_engine_loop: bool = True,
```

File: vllm/engine/arg_utils.py [hunk 2]
```diff
@@ -183,4 +184,8 @@ def add_cli_args(
         parser.add_argument('--disable-log-requests',
                             action='store_true',
                             help='disable logging requests')
+        parser.add_argument('--max_log_len',
+                            type=int,
+                            default=None,
+                            help='max prompt/token being printed in log')
```
Comment (zhuohan123) [inline vllm/engine/arg_utils.py:190]: Nit: 
```suggestion
        parser.add_argument('--max-log-tokens',
                            type=int,
                            default=None,
                            help='max number of prompt tokens being printed in log')
```
Please rename `max_log_len` to `max_log_tokens` in other places. Thanks!

**Review by zhuohan123 — CHANGES_REQUESTED** [review]
Thank you for your contribution! Left some small comments about naming and style

File: vllm/engine/arg_utils.py (hunk 2 above)
Comment (leiwen83) [reply vllm/engine/arg_utils.py]: Done for the rename

**Review by zhuohan123 — APPROVED** [review]
LGTM! Fixed some minor issues. Thank you for your contribution!

**Comment by zhuohan123** [comment]
Actually, I changed the name back to "max_log_len" since for prompt string, it limits the number of characters instead of tokens.

**Pull request merged**